utils/emptyChecker.js

```diff
@@ -0,0 +1,13 @@
+// Empty object checker
```

Comment:
do we need to disable those eslint rules?

Reply:
If you think we should then I'll disable them on the next PR.